src/modules/governance/pages/daoMembersPage/daoMembersPage.tsx

```diff
@@ -1,5 +1,6 @@
 import { daoOptions } from '@/shared/api/daoService';
 import { Page } from '@/shared/components/page';
+import { redirect } from '@/shared/lib/nextNavigation';
```

Comment:
```suggestion
import { redirect } from 'next/navigation';
```

`tsconfig.json` has configuration which maps `next/navigation -> ./src/shared/lib/nextNavigation`